example/topics/main.go

```diff
@@ -3,9 +3,11 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// The simple command demonstrates a simple functionality which
-// prompts the user for a GitHub topic and lists all the public
-// organization memberships of the specified topic.
+// The simple command demonstrates the functionality that
+// prompts the user for a GitHub topic and lists all the entities
+// that are related to the specified topic or subject.
+//
+// [START main.go]
```

Comment:
what's the purpose of these START and END blocks?

Reply:
@willnorris I habitually use that block format for documentation purposes. You can delete those [START and [END  comments.  The changes were only in comment blocks. 
Your project is impressive in its scope. Thank you for the good work!